packages/ui/src/composites/wui-list-select/index.ts

```diff
@@ -0,0 +1,63 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { globalStyles } from '../../utils/ThemeUtil'
+import '../../components/wui-text'
+import '../wui-wallet-image'
+import '../wui-all-wallets-image'
+import styles from './styles'
+import type { IWalletImage } from '../../utils/TypesUtil'
+
+@customElement('wui-list-select')
+export class WuiListSelect extends LitElement {
+  public static styles = [globalStyles, styles]
+
+  // -- state & properties ------------------------------------------- //
+  @property({ type: Array }) public walletImages?: IWalletImage[] = []
+
+  @property() public imageSrc? = ''
+
+  @property() public listName = ''
+
+  @property() public listStatus?: string
+
+  @property({ type: Boolean }) public showAllWallets = false
```

Comment:
There is no need to for this as we can check if `walletImages` is undefined i.e. further down below:

```
if (this.showAllWallets && this.walletImages) {
 throw new Error('Either showAllWallets or walletImages can be defined, not both')
}

// ...

{this.showAllWallets && <wui-all-wallet....}
{this.imageSrc && <wui-wallet-imag...}
```